samples/node/routes/weather.js

```diff
@@ -29,8 +29,11 @@ exports.testAuth = function(req, res) {
   res.status(200).send();
 }
 
+let lastReported = {};
+
 exports.reportWeather = function (req, res) {
     console.log(`Reporting temperature of ${req.body.temperature} for ${req.body.zip}`);
+    lastReported[req.body.zip] = req.body.temperature;
```

Comment:
Having a little state on the connector makes it more useful to test with.  Sometimes in development, the client errors and doesn't handle rest call responses properly (the button gets stuck in-progress) and you aren't sure if the call made it through the server.

This allows you to fetch the card again and see different state instead of chasing down connector or server logs.